scripts/apply_style.py

```diff
@@ -290,14 +318,18 @@ def main():
     format_python(args)
     output('Formatting CMake ...')
     format_language(args, **SUPPORTED_LANGUAGES['cmake'])
+
+    output('Checking header include guards ...')
+    fix_missing_include_guards()
+
     output('Done.')
 
     if args.diff:
         diff = get_diff()
         if diff:
-            print('*' * 80)
-            print(diff)
-            print('*' * 80)
+            output('*' * 80)
```

Comment:
One for the todo list - we can replace `output` for `print` provided we run the script with `PYTHONUNBUFFERED=1`

Reply:
Doesn't that mean that our way is therefore safer? I doubt the performance improvement would be measurable - I expect this script is limited by disk access by a long way.